code/game/turfs/space/transit.dm

```diff
@@ -64,8 +64,8 @@
 			_y = min
 
 	var/turf/T = locate(_x, _y, _z)
-	AM.loc = T
-	AM.newtonian_move(dir)
```

Comment:
will people still drift without this?

Reply:
Yes, forced move calls Moved and that calls this